Assets/JS/app.js

```diff
@@ -0,0 +1,35 @@
+const addProductToCartTable = (event) => {
+    event.preventDefault();
+    const productData = [
+        {
+            productName: document.getElementById('productName').value,
+            productPrice: document.getElementById('productPrice').value,
+            productCategory: document.getElementById('productCategory').value,
+            productURL: document.getElementById('productPhotoSrc').value
+        }
+    ];
+    const cartTable = document.getElementById('cartTable');
+    const newRow = document.createElement('tr');
+    newRow.innerHTML = `
+        <td>${productData[0].productName}</td>
+        <td>${productData[0].productPrice}</td>
+        <td>${productData[0].productCategory}</td>
+        <td><img src="${productData[0].productURL}" alt="${productData[0].productName}" width="100"></td>
+        <td><button id="deleteFromTable" onclick="deleteProductFromCartTable()">Delete</button></td>
+    `;
+    cartTable.appendChild(newRow);
+    clearForm();
+}
+
+const clearForm = () => {
+    document.getElementById('productName').value = '';
+    document.getElementById('productPrice').value = '';
+    document.getElementById('productCategory').value = 'select';
```

Comment:
the proper way to clear the select is to set the value to ''. notice that it does "clear" on submission, because you're setting it to a value that doesn't exist, so it becomes blank, without the "Select" label